src/components/Blocking/BlockDomain.js

```diff
@@ -5,6 +5,7 @@ import {StyleSheet, Text, View} from 'react-native';
 import ItemListLarge from './ItemListLarge';
 import {colors} from '../../utils/colors';
 import {fonts} from '../../utils/fonts';
+import { COLORS } from '../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `COLORS` from `'../../utils/theme'` is added, but the previous import `{colors} from '../../utils/colors'` is not removed. If `COLORS` is replacing `colors`, then the unused import should be removed to keep the code clean and avoid confusion.
```diff
- import {colors} from '../../utils/colors';
+ // Removed as COLORS is now being used
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
@agit26milan masih ada `import {colors} ...` better dihapus aja mas import ini karena tidak dipakai